pose_diff/Test/core/test_method.py

```diff
@@ -44,3 +49,33 @@ def test_video():
     plt.show()
 
     print("Done")
+
+def test_video2():
+    '''
+    Test result of video
+
+    Todo
+        1. cutframes의 개수가 75보다 낮은 것들은 작동하지 않는다. 따라서 제외한다. 왜 작동하지 않는지도 알아내야 한다.
+        2. 에러값을 주면 오류가 난다. 원인: run.py score_angle부분에서 에러가 발생한다.
+    '''
+    base_folder = 'pose_diff/Test/core/test_data2'
+    graph_numpy = np.load(os.path.join(base_folder, 'graph.npy'))
+    recollected_frames = graph_numpy[-1]
+    print('length of video: %d' % len(recollected_frames))
+    for idx, part in enumerate(recollected_frames[3:4]):
```

Comment:
Run.Video does not work and just go over when frame cut < 75. Used frame is daan's

Reply:
It occurs because of frame_cut. So  it would be upgrade the framecut.py in core.
Please wait with a little patience. I'll figure it out soon.

Reply:
I am okay if you say so, but you should add this as issue for release 1.0.1